documentation/wallet.adoc

```diff
@@ -381,9 +448,64 @@ IMPORTANT: The unsealed private key is neither stored nor cached. It is utilized
 To sign a message, the content undergoes hashing using the Blake3 algorithm. The hashed value is subsequently signed using the ED25519 signature scheme. This function ensures the message signature process aligns with the Massa Labs cryptography standard.
 
 == Non-Functional Requirements
+=== Performance Requirements:
+** 1. Response Time:**
+The Massa Wallet module has a responsive user interface, with an
+ average response time for any user action (e.g., sending funds, 
+checking balances), not exceeding 2 seconds.
+
+**2. Scalability:**
+The module is able to handle a growing number of transactions and users by design. 
+All wallet components, including the API, run locally on the user's computer, so each user have API instance.
+*The scalability bottleneck is on the node side; if 10,000 users
+request to do an operation at the same time on the same node,
+it may be a problem for Massa Wallet users.*
+
+**3. Resource Utilization:**
+The module is resource-efficient, consuming no more than 100 MB of RAM and 1% of CPU utilization 
+during typical usage. During periods of inactivity or when the wallet is idle, resource utilization is near 0%.
+
+=== Security Requirements:
+
+**1. Data Encryption:**
+All sensitive user data, such as private keys and transaction history, are encrypted both at 
+rest and during transmission. 
+
+**name Industry-standard encryption protocols, and link to their usage.**
+
+**2. Authentication and Authorization:**
+Users are required to authenticate using password, or follow the recovery process to restore wallet accounts. 
+
+**Define rules for password authorization mechanism here** 
```

Comment:
There is no authorisation and authentication at wallet level. password is used to access the private key in clear.